src/amuse/units/quantities.py

```diff
@@ -12,6 +12,12 @@
 from amuse.units.si import none
 from amuse.units.core import zero_unit
 
+try:
+    import astropy.units
+    from amuse.units import units
+    astropy_available = True
```

Comment:
usually this kind of variable (global in the file is denoted by some combination of capitals and underscores...), so:
__HAS_ASTROPY__=True

Reply:
done